front/pages/w/[wId]/builder/assistants/[aId]/index.tsx

```diff
@@ -177,6 +186,8 @@ export const getServerSideProps: GetServerSideProps<{
       };
     }
   }
+  assert(BUILDER_FLOWS.includes(context.query.flow as BuilderFlow));
```

Comment:
See the other comment. Let's not assert and validate + default instead (here you prevent someone from going to the page by simple URL)